website/src/data/tweets/index.tsx

```diff
@@ -0,0 +1,393 @@
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+
+import {Props as Tweet} from '../../components/Tweet';
+
+export interface TweetItem extends Tweet {
+  showOnHomepage: boolean;
+}
+
+const TWEETS: TweetItem[] = [
+  {
+    url: 'https://twitter.com/acemarke/status/1452725153998245891',
+    handle: 'acemarke',
+    name: 'Mark Erikson',
+    date: 'Oct 26, 2021',
+    avatar:
+      'https://pbs.twimg.com/profile_images/842582724737163264/tFKLiJI5_400x400.jpg',
+    content: (
+      <>
+        We&apos;ve been using Docusaurus for all the Redux org docs sites for
+        the last couple years, and it&apos;s great! We&apos;ve been able to
+        focus on content, customize some presentation and features, and It Just
+        Works.
+      </>
+    ),
+    showOnHomepage: true,
+  },
+  {
+    url: 'https://twitter.com/arcanis/status/1351620354561732608',
+    handle: 'arcanis',
+    name: 'Maël',
+    date: 'Jan 20, 2021',
+    avatar:
+      'https://pbs.twimg.com/profile_images/1311259425949261825/7hPZqoJd_400x400.jpg',
+    content: (
+      <>
+        I&apos;ve used Docusaurus for two websites this year, and I&apos;ve been
+        very happy about the v2. Looks good, and simple to setup.
+      </>
+    ),
+    showOnHomepage: true,
+  },
+  {
+    url: 'https://twitter.com/maxlynch/status/1375113166007455748',
+    handle: 'maxlynch',
+    name: 'Max Lynch',
+    date: 'Mar 25, 2021',
+    avatar:
+      'https://pbs.twimg.com/profile_images/1318970727173885953/bln98FNj_400x400.jpg',
+    content: (
+      <>
+        Docusaurus v2 doubles as a really nice little static site generator tool
+        for content-focused sites, love it 👏
+      </>
+    ),
+    showOnHomepage: true,
+  },
+  {
+    url: 'https://twitter.com/supabase/status/1328960757149671425',
+    handle: 'supabase',
+    name: 'Supabase',
+    date: 'Nov 18, 2020',
+    avatar:
+      'https://pbs.twimg.com/profile_images/1397471927132844033/jN-wuufb_400x400.jpg',
+    content: (
+      <>
+        We&apos;ve been using V2 since January and it has been great - we spend
+        less time building documentation and more time building 🛳
+        <br />
+        <br />
+        Thanks @docusaurus team 🦖
+      </>
+    ),
+    showOnHomepage: true,
+  },
+  {
+    url: 'https://twitter.com/paularmstrong/status/1387059593373700100',
+    handle: 'paularmstrong',
+    name: 'Paul Armstrong',
+    date: 'Apr 27, 2021',
+    avatar:
+      'https://pbs.twimg.com/profile_images/823614982394769408/C4KgET17_400x400.jpg',
+    content: (
+      <>
+        Continue to be impressed and excited about Docusaurus v2 alpha releases.
+        Already using the sidebar items generator to help monorepo workspace
+        devs create their own doc pages without any configuration needed.
+      </>
+    ),
+    showOnHomepage: true,
```

Comment:
I assume we will have a separate "testimonials" page in the future?

Reply:
Yes one page with more categories of Tweets according to [this](https://github.com/facebook/docusaurus/pull/6426#issuecomment-1018644968)